include/vapor/RenderParams.h

```diff
@@ -484,46 +497,89 @@ class PARAMS_API RenderParams : public ParamsBase {
     static const string CustomHistogramRangeTag;
     static const string LightingEnabledTag;
 
-    //! If a renderer supports rotation about a point of origin,
-    //! this string identifies the parameter for the origin's
+    //! If a renderer supports rotation about a point of origin (IE - Slice and Contour),
+    //! this tag identifies the parameter for the origin's
```

Comment:
Question for @stasj: how does the auto-python generation code handle parameters that don't have explicit setters/getters? I.e. does it generate Python setters and getters, or generate a generic Python function that accepts a tag? If the former, will this documentation make sense for Python users?

Reply:
It generates python setters and getters. The setters/getters will include the tag documentation as well as specify the data type that the tag expects. We could simply omit referring to it as a "tag" if that is confusing.

Reply:
I think it is more important for the documentation to be clear for the Python users. The GUI users get a lot of information by context. So, yes, I agree omitting tag would be better. E.g. something like:

"This parameter identifies the origin's location..."

would probably work for both GUI and Python users.